.env.example

```diff
@@ -78,3 +78,7 @@ ENABLE_WW1_FULLTEXT_SEARCH=0
 # Name given to you by Disqus. https://help.disqus.com/en/articles/1717111-what-s-a-shortname
 # If absent, Disqus comments will be disabled.
 DISQUS_SHORTNAME=shortname-here
+
+# Enable authentication for User galleries (XX), enabled by default
+ENABLE_AUTH=1
+OAUTH_URL=https://auth.europeana.eu/auth/realms/europeana/protocol/openid-connect
```

Comment:
Since the portal needs to run on https, a more descriptive set of instructions would be useful, in order to set up the portal successfully.